stacks/base.py

```diff
@@ -0,0 +1,32 @@
+"""Base stack with R2 backend."""
+
+from cdktf import S3Backend
+
+from helicopyter import HeliStack
+
+
+class BaseStack(HeliStack):
```

Comment:
Why is `BaseStack` specific to R2? Imo that doesn't sound like a "base" to me, more of a child

Reply:
I wanted to try out R2. (It works!)

I'd like Helicopyter to support a base stack class, which consistently defines the backend and makes any common providers easy, and then different child stack classes for onsite and cloud infrastructure which deal with their respective providers. But that doesn't mean the example here needs to follow the same pattern. I'll try to rename in a follow up.